Gemfile

```diff
@@ -37,3 +37,4 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 gem "rspec-rails", group: [:development, :test]
+gem 'faker','~>3.1.1'
```

Comment:
Try to keep the gems in alphabeitcal order.
A standard rails Gemfile should look like this:

```ruby
source 'https://rubygems.org'
git_source(:github) { |repo| "https://github.com/#{repo}.git" }
ruby '<ruby-version>'

gem rails, '<rails-version>'

# <common gems for all environments, with versions, in alphabetical order>

group :development do
  # <gems for development in alphabetical order>
end

group :development, :test do
  #<gems for development and test in alphabetical order>
end

group :test do
  # <gems for test in alphabetical order>
end
```

Reply:
also try to keep the Gemfile without comments, unless absolutely necessary e.g. a gem has a specific version because of some compatibility issues